source/manage/client-proxysql-metrics.rst

```diff
@@ -34,14 +34,14 @@ The output of this command may look as follows:
      Service name: ubuntu-proxysql
 
 Beside positional arguments shown above you can specify service name and
-service address with the following flags: ``--service-name``, ``--host`` (the
-hostname or IP address of the service), and either ``--port`` (the port number of the
-service) or ``--socket`` (the UNIX socket path). If both flag and positional argument are present, flag gains higher
-priority. Here is the previous example modified to use these flags for both port and socket connections::
+service address with the following flags: ``--service-name``, and ``--host`` (the
+hostname or IP address of the service) and ``--port`` (the port number of the
+service), or ``--socket`` (the UNIX socket path). If both flag and positional argument are present, flag gains higher
+priority. Here is the previous example modified to use these flags for both host/port or socket connections::
 
      pmm-admin add proxysql --username=pmm --password=pmm --service-name=my-new-proxysql --host=127.0.0.1 --port=3306 
 
-     pmm-admin add proxysql --username=pmm --password=pmm --service-name=my-new-proxysql --host=127.0.0.1 --socket=/var/run/unix_socket
+     pmm-admin add proxysql --username=pmm --password=pmm --service-name=my-new-proxysql --socket=/var/run/unix_socket
```

Comment:
Socket option for  ProxySQL not yet   Tested and merged  so it'a better to keep it   until it's done on DEV  and QA side

Reply:
Then should this not be merged into 2.6.0? This is a conflict between it and master.